test/runtimeconfig.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	setup_test
+}
+
+function teardown() {
+	cleanup_test
+}
+
+@test "crictl info prints out the crio runtime config" {
+	start_crio
+	info_json=$(crictl info -o json)
+
+	jq -e '.config.crio' <<< "$info_json"
+
+	default_runtime=$(jq -r ".config.crio.DefaultRuntime" <<< "$info_json")
+	expected=$(get_runtime)
+	[[ "${default_runtime}" == "$expected" ]]
+
+	jq -e ".config.crio.Runtimes.${default_runtime}" <<< "$info_json"
+
+	runtime_type=$(jq -r ".config.crio.Runtimes.${default_runtime}.RuntimeType" <<< "$info_json")
+	[[ "${runtime_type}" == "oci" ]]
+
+	runtime_binary_path=$(jq -r ".config.crio.Runtimes.${default_runtime}.RuntimePath" <<< "$info_json")
+	# shellcheck disable=SC2153
```

Comment:
We no longer have to disable shellcheck, do we?

Reply:
This is still required actually